src/_gettsim/transfers/arbeitsl_geld_2/kindergelduebertrag.py

```diff
@@ -104,6 +104,7 @@ def kindergeld_zur_bedarfsdeckung_m(
 def _diff_kindergeld_kindbedarf_m(
     _arbeitsl_geld_2_eink_ohne_kindergeldübertrag_m_bg: float,
     arbeitsl_geld_2_regelbedarf_m_bg: float,
+    kinderwohngeld_m: float,
```

Comment:
why cannot we use `wohngeld_vor_vorrang_check_m` directly here?

As Kinderwohngeld isn't a term in the law (if I understood you correctly) I would try to avoid it.

Reply:
The `if kind and eigenbedarf_gedeckt else 0.0` is central here. `eigenbedarf_gedeckt` is our way of saying that the child actually receives Wohngeld (so it anticipates the Vorrangprüfung to not create a cycle in the DAG). If we use `wohngeld_vor_vorrang_check_m` directly when calculating the Kindergeldübertrag, we make errors if the child is not actually eligible for Wohngeld (because Eigenbedarf is not satisfied).

I agree that it would be ideal to avoid modelling KiWoG explitcitly (because it's just Wohngeld) but this does work only if we compute Wohngeld/ALG2 for parents and children separately, as mentioned [here](https://github.com/iza-institute-of-labor-economics/gettsim/pull/753#issuecomment-2098457626).

Reply:
> if we compute Wohngeld/ALG2 for parents and children separately

Indeed that's how I had been thinking about it all the time. No other way of getting to `eigenbedarf_gedeckt` and we cannot expect users to supply that (outside of example households, that is).

Reply:
I thought you could just use `wohngeld_vor_vorrang_check_m` as input variable here and then subtract only if it is a kind.  `eigenbedarf_gedeckt` shouldn't matter as out is set to 0 whenever  `eigenbedarf_gedeckt == False` anyway.

But this isn't important in the very short term.

Reply:
Ah you're right, we could do that!